### ISSUE TITLE
Unable to remove a piece of track

### ISSUE THREAD

**Issue by NSTrains** [OpenLoco/OpenLoco#2801]
Unable to remove a piece of track or put a signal, overhead wires on it. When trains try to go over the piece of track, they stop and turn around. For  tempory solution I built around the track, but every so often I get a crash, either it gives me an error and dump file, or it doesn't show anything at all. This happens on one save game that I started to play.

If anyone wants to try to reproduce this, place a piece of track down and try to remove it, or place a signal, overhead wires on the track. 

When I try to remove the track, it says "Can't build railroad track" and "Can't build signals here"

**Comment by AaronVanGeffen** [comment]
Could you attach a save game? (You'll need to zip it for GitHub to allow it.)

The issue itself reminds me of #928 as well.

**Comment by NSTrains** [comment]
it says failed to upload attachment

**Comment by AaronVanGeffen** [comment]
As I said, you'll need to zip it for GitHub to allow it.

**Comment by NSTrains** [comment]
I tried a zip file, but it fails to upload

**Comment by NSTrains** [comment]
<img width="827" alt="Image" src="https://github.com/user-attachments/assets/f4f4d63f-1822-42c6-9a47-3720748132aa" />

**Comment by LeftofZen** [comment]
Strange, maybe you can try uploading it to some online file sharing service?

**Comment by NSTrains** [comment]
See if you can get it on here https://drive.google.com/file/d/1yRWPJuCLLZ0mwCZMmfLCekl5e-slEXw4/view?usp=sharing

**Comment by LeftofZen** [comment]
Got it, thanks. 24.11 crashes when it tries to load this save :( I'll investigate

**Comment by NSTrains** [comment]
Does it give you any missing object error?

**Comment by LeftofZen** [comment]
No, it just crashes in a paint function about a null land element on tile [87, 42]. I see a lot of packed objects in the file though. If I bypass that it has a null tree element on the tile. Bypassing that, null wall element. After that, null industry elements. Null road elements. About 10 more null checks I added and the game doesn't render *anything* now. I also get this in the console:
![Image](https://github.com/user-attachments/assets/92a5b3b0-74d3-4129-9544-a7ce4599550d)


How are you playing this save? What version of openloco are you using?

**Comment by NSTrains** [comment]
Try replacing the object with this one https://drive.google.com/file/d/1lGN5LSIxRQ2U7Ah8rkDaDrHt6brUd2jg/view?usp=sharing

**Comment by LeftofZen** [comment]
It just failed on the next packed object. I know there is some issue with packed object loading in openloco at the moment, so this is probably what is preventing me from loading the save. What version of openloco are you using currently?

**Comment by NSTrains** [comment]
Im using 24.11

**Comment by duncanspumpkin** [comment]
> It just failed on the next packed object. I know there is some issue with packed object loading in openloco at the moment, so this is probably what is preventing me from loading the save. What version of openloco are you using currently?
> 
> Also bizarrely, the game crashes in a paint function for tile elements before the map is even loaded, which seems wrong. Not sure if its the issue here or not.

That is expected due to the packed objects. Open the save in 24.04 or something old to unpack the objects then it should load fine

**Comment by luk3ZPL** [comment]
> <img alt="Image" width="827" src="https://private-user-images.githubusercontent.com/53015176/396686392-f4f4d63f-1822-42c6-9a47-3720748132aa.PNG?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzQ4NDcxMjAsIm5iZiI6MTczNDg0NjgyMCwicGF0aCI6Ii81MzAxNTE3Ni8zOTY2ODYzOTItZjRmNGQ2M2YtMTgyMi00MmM2LTlhNDctMzcyMDc0ODEzMmFhLlBORz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEyMjIlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMjIyVDA1NTM0MFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTVhMjE3M2Y3ZTQwMzZhNjIyZmFlNmQ5NDc5NTcxMjk3OWQzODhiMzVjNDk3ZjA1NDkzNjczYmI0YmUxZTliNjQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.9a40XwuQo8n-Jxx6TRDwcxHX-bGv2mtCF9KvLFk8rGY">

JavaScript disabled in the browser (possibly uBlock/AdBlock).

**Comment by NSTrains** [comment]
> > <img alt="Image" width="827" src="https://private-user-images.githubusercontent.com/53015176/396686392-f4f4d63f-1822-42c6-9a47-3720748132aa.PNG?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzQ4NDcxMjAsIm5iZiI6MTczNDg0NjgyMCwicGF0aCI6Ii81MzAxNTE3Ni8zOTY2ODYzOTItZjRmNGQ2M2YtMTgyMi00MmM2LTlhNDctMzcyMDc0ODEzMmFhLlBORz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEyMjIlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMjIyVDA1NTM0MFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTVhMjE3M2Y3ZTQwMzZhNjIyZmFlNmQ5NDc5NTcxMjk3OWQzODhiMzVjNDk3ZjA1NDkzNjczYmI0YmUxZTliNjQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.9a40XwuQo8n-Jxx6TRDwcxHX-bGv2mtCF9KvLFk8rGY">
> 
> JavaScript disabled in the browser (possibly uBlock/AdBlock).

error 404 when java script is enabled and adblock disabled
